src/tsystemglobal.cpp

```diff
@@ -22,9 +22,11 @@
 #include <TWebApplication>
 
 constexpr auto DEFAULT_SYSTEMLOG_LAYOUT = "%d %5P %m%n";
-constexpr auto DEFAULT_SYSTEMLOG_DATETIME_FORMAT = "yyyy-MM-ddThh:mm:ss";
+constexpr auto DEFAULT_SYSTEMLOG_DATETIME_FORMAT = "yyyy-MM-dd hh:mm:ss";
 constexpr auto DEFAULT_ACCESSLOG_LAYOUT = "%h %d \"%r\" %s %O%n";
-constexpr auto DEFAULT_ACCESSLOG_DATETIME_FORMAT = "yyyy-MM-ddThh:mm:ss";
+constexpr auto DEFAULT_ACCESSLOG_DATETIME_FORMAT = "yyyy-MM-dd hh:mm:ss";
+constexpr auto DEFAULT_QUERYLOG_LAYOUT = "%d [%t] %m%n";
```

Comment:
This is different in comparison to `SqlQueryLog.Layout` in application.ini.

I also noticed that you also modified the DATETIME format with this commit. If on purpose, it deserve a mention in the changelog. :-)